.cicd/package-builder.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+set -eo pipefail
+cd $( dirname "${BASH_SOURCE[0]}" ) # Ensure we're in the .cicd dir
+. ./.helpers
```

Comment:
Where is this library used in this script? Seems to me like it doesn't need to be included here.

Reply:
Good catch, you're right